### ISSUE TITLE
🐛[BUG]npm run openapi 报错'@umijs\openapi\dist\mockGenerator.js:Unexpected token '.

### ISSUE THREAD

**Issue by birdycn** [ant-design/ant-design-pro#8046]
```
Unexpected token '.'
D:\node_modules\@umijs\openapi\dist\mockGenerator.js:151
                const conte = methodConfig?.tags?.join('/') ||      
                                           ^

SyntaxError: Unexpected token '.'
    at compileFunction (<anonymous>)
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)      
    at Module._compile (D:\node_modules\pirates\lib\index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Object.newLoader [as .js] (D:\node_modules\pirates\lib\index.js:104:7)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)    at Module.require (internal/modules/cjs/loader.js:887:19)       
    at require (internal/modules/cjs/helpers.js:74:18)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ant-design-pro@5.0.0-beta.2 openapi: `umi openapi`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ant-design-pro@5.0.0-beta.2 openapi script.  
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\xxx\AppData\Roaming\npm-cache\_logs\2021-02-04T12_22_29_766Z-debug.log
```

**Comment by chenshuai2144** [comment]
升级下node的版本 我们用了node14

**Comment by birdycn** [comment]
> 升级下node的版本 我们用了node14

升级到14还是会报错

```
[openAPI]: 💺 将 Swagger 转化为 openAPI
[GenSDK] gen service param error: TypeError: Cannot read property 'replace' of undefined
    at Object.stripDot (D:\\node_modules\@umijs\openapi\dist\util.js:183:16)
    at D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:254:59
    at Array.map (<anonymous>)
    at D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:241:18
    at Array.map (<anonymous>)
    at ServiceGenerator.getServiceTP (D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:234:14)
    at ServiceGenerator.genFile (D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:210:14)
    at generateService (D:\\node_modules\@umijs\openapi\dist\index.js:60:22)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Cannot read property 'replace' of undefined
TypeError: Cannot read property 'replace' of undefined
    at Object.stripDot (D:\\node_modules\@umijs\openapi\dist\util.js:183:16)
    at D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:254:59
    at Array.map (<anonymous>)
    at D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:241:18
    at Array.map (<anonymous>)
    at ServiceGenerator.getServiceTP (D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:234:14)
    at ServiceGenerator.genFile (D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:210:14)
    at generateService (D:\\node_modules\@umijs\openapi\dist\index.js:60:22)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ant-design-pro@5.0.0-beta.2 openapi: `umi openapi`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ant-design-pro@5.0.0-beta.2 openapi script.  
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\xxx\AppData\Roaming\npm-cache\_logs\2021-02-04T16_33_16_298Z-debug.log```

**Comment by baiheinet** [comment]
升级试试 "@umijs/plugin-openapi": "^1.0.4",

**Comment by birdycn** [comment]
> 升级试试 "@umijs/plugin-openapi": "^1.0.4",

已经是"@umijs/plugin-openapi": "^1.0.4",昨晚才拉的包

**Comment by birdycn** [comment]
@chenshuai2144 您好，升级到14还是会报错

```
[openAPI]: 💺 将 Swagger 转化为 openAPI
[GenSDK] gen service param error: TypeError: Cannot read property 'replace' of undefined
    at Object.stripDot (D:\\node_modules\@umijs\openapi\dist\util.js:183:16)
    at D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:254:59
    at Array.map (<anonymous>)
    at D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:241:18
    at Array.map (<anonymous>)
    at ServiceGenerator.getServiceTP (D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:234:14)
    at ServiceGenerator.genFile (D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:210:14)
    at generateService (D:\\node_modules\@umijs\openapi\dist\index.js:60:22)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
Cannot read property 'replace' of undefined
TypeError: Cannot read property 'replace' of undefined
    at Object.stripDot (D:\\node_modules\@umijs\openapi\dist\util.js:183:16)
    at D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:254:59
    at Array.map (<anonymous>)
    at D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:241:18
    at Array.map (<anonymous>)
    at ServiceGenerator.getServiceTP (D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:234:14)
    at ServiceGenerator.genFile (D:\\node_modules\@umijs\openapi\dist\serviceGenerator.js:210:14)
    at generateService (D:\\node_modules\@umijs\openapi\dist\index.js:60:22)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ant-design-pro@5.0.0-beta.2 openapi: `umi openapi`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ant-design-pro@5.0.0-beta.2 openapi script.  
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\xxx\AppData\Roaming\npm-cache\_logs\2021-02-04T16_33_16_298Z-debug.log```
```

**Comment by HTMHell** [comment]
I've had this error, and updating node to 14 solved it for me

**Comment by chenshuai2144** [comment]
i will build it

**Comment by chenshuai2144** [comment]
fix by https://github.com/chenshuai2144/openapi2typescript/commits/main

**Comment by birdycn** [comment]
> fix by [chenshuai2144/openapi2typescript@be77a67](https://github.com/chenshuai2144/openapi2typescript/commit/be77a67a9656bb8679259236fb768352f0289e88)

npm包版本还是1.0.4
